docs/gen-ai/non-normative/examples-llm-calls.md

```diff
@@ -296,8 +296,8 @@ They are likely to be siblings if there is an encompassing span.
 | `gen_ai.usage.output_tokens`    | `17`                                        |
 | `gen_ai.usage.input_tokens`     | `47`                                        |
 | `gen_ai.response.finish_reasons`| `["tool_calls"]`                            |
-| `gen_ai.input.messages`         | <span id="gen-ai-input-messages-tool-call-span-1">`gen_ai.input.messages` value</span> |
```

Comment:
@Cirilla-zmh does this generated automatically? https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#2-update-the-markdown-files